homeassistant/components/shelly/utils.py

```diff
@@ -98,7 +98,7 @@ def get_block_entity_name(
     channel_name = get_block_channel_name(device, block)
 
     if description:
-        return f"{channel_name} {description}"
+        return f"{channel_name} {description}".capitalize()
```

Comment:
```suggestion
        return f"{channel_name} {description.lower()}"
```
Wouldn't it be better just to lowercase the description and leave the name as it is? afterall the name arrives from the user/device and we don't want to change it

Reply:
Good point, thanks.

Reply:
While testing I noticed we need to fix it for Gen2:
https://github.com/home-assistant/core/blob/8a72aadd5529f05351497a31e6282b5aea0ab31b/homeassistant/components/shelly/utils.py#L329

Reply:
I missed that, thanks.